README.md

```diff
@@ -53,28 +103,40 @@ subprojects {
 
 ### IntelliJ Idea Only Plugin
 
-(This plugin is automatically applied by the `ktlint` plugin.)
+**Note:** This plugin is automatically applied by the main `KtLint` plugin.
```

Comment:
Lowercase "k" I think.

Reply:
seems everything in name should be lowercase